👀 Netlify is failing. Need to investigate this

👀 Netlify is failing. Need to investigate this
cc @IvanGoncharov

_Originally posted by @saihaj in https://github.com/graphql/swapi-graphql/pull/182#issuecomment-714849611_

@saihaj It looks like it caused by Netlify not running `download` command:
<details>
  <summary>Netlify log</summary>
<pre>
9:57:23 PM: Build ready to start
9:57:25 PM: build-image version: b0258b965567defc4a2d7e2f2dec2e00c8f73ad6
9:57:25 PM: build-image tag: v3.4.1
9:57:25 PM: buildbot version: db1c11c310036491f94882c0e310ea68c81c7e60
9:57:25 PM: Fetching cached dependencies
9:57:25 PM: Starting to download cache of 243.1MB
9:57:27 PM: Finished downloading cache in 2.543409892s
9:57:27 PM: Starting to extract cache
9:57:35 PM: Finished extracting cache in 7.6616387s
9:57:35 PM: Finished fetching cache in 10.258412111s
9:57:35 PM: Starting to prepare the repo for build
9:57:35 PM: Preparing Git Reference pull/182/head
9:57:37 PM: Different build command detected, going to use the one specified in the Netlify configuration file: 'npm run build' versus 'yarn build' in the Netlify UI
9:57:37 PM: Starting build script
9:57:37 PM: Installing dependencies
9:57:37 PM: Python version set to 2.7
9:57:38 PM: Started restoring cached node version
9:57:40 PM: Finished restoring cached node version
9:57:41 PM: v10.22.1 is already installed.
9:57:42 PM: Now using node v10.22.1 (npm v6.14.6)
9:57:42 PM: Started restoring cached build plugins
9:57:42 PM: Finished restoring cached build plugins
9:57:42 PM: Attempting ruby version 2.6.2, read from environment
9:57:43 PM: Using ruby version 2.6.2
9:57:43 PM: Using PHP version 5.6
9:57:43 PM: 5.2 is already installed.
9:57:43 PM: Using Swift version 5.2
9:57:43 PM: Started restoring cached node modules
9:57:43 PM: Finished restoring cached node modules
9:57:44 PM: Started restoring cached go cache
9:57:45 PM: Finished restoring cached go cache
9:57:45 PM: Installing Go version 1.12
9:57:45 PM: unset GOOS;
9:57:45 PM: unset GOARCH;
9:57:45 PM: export GOROOT='/opt/buildhome/.gimme_cache/versions/go1.12.linux.amd64';
9:57:45 PM: export PATH="/opt/buildhome/.gimme_cache/versions/go1.12.linux.amd64/bin:${PATH}";
9:57:45 PM: go version >&2;
9:57:45 PM: export GIMME_ENV='/opt/buildhome/.gimme_cache/env/go1.12.linux.amd64.env';
9:57:45 PM: go version go1.12 linux/amd64
9:57:45 PM: Installing missing commands
9:57:45 PM: Verify run directory
9:57:46 PM: ​
9:57:46 PM: ┌─────────────────────────────┐
9:57:46 PM: │        Netlify Build        │
9:57:46 PM: └─────────────────────────────┘
9:57:46 PM: ​
9:57:46 PM: ❯ Version
9:57:46 PM:   @netlify/build 5.0.0
9:57:46 PM: ​
9:57:46 PM: ❯ Flags
9:57:46 PM:   deployId: 5f8b3e935c42d200089c0d21
9:57:46 PM:   mode: buildbot
9:57:46 PM: ​
9:57:46 PM: ❯ Current directory
9:57:46 PM:   /opt/build/repo
9:57:46 PM: ​
9:57:46 PM: ❯ Config file
9:57:46 PM:   /opt/build/repo/netlify.toml
9:57:46 PM: ​
9:57:46 PM: ❯ Context
9:57:46 PM:   deploy-preview
9:57:47 PM: ​
9:57:47 PM: ┌────────────────────────────────────┐
9:57:47 PM: │ 1. build.command from netlify.toml │
9:57:47 PM: └────────────────────────────────────┘
9:57:47 PM: ​
9:57:47 PM: $ npm run build
9:57:47 PM: > swapi-graphql@ build /opt/build/repo
9:57:47 PM: > rimraf lib && babel src --ignore __tests__,public --out-dir lib/ && npm run build:lambda
9:57:48 PM: src/api/index.js -> lib/api/index.js
9:57:48 PM: src/api/local.js -> lib/api/local.js
9:57:48 PM: src/schema/apiHelper.js -> lib/schema/apiHelper.js
9:57:48 PM: src/schema/commonFields.js -> lib/schema/commonFields.js
9:57:48 PM: src/schema/connections.js -> lib/schema/connections.js
9:57:48 PM: src/schema/index.js -> lib/schema/index.js
9:57:48 PM: src/schema/relayNode.js -> lib/schema/relayNode.js
9:57:48 PM: src/schema/types/film.js -> lib/schema/types/film.js
9:57:48 PM: src/schema/types/person.js -> lib/schema/types/person.js
9:57:48 PM: src/schema/types/planet.js -> lib/schema/types/planet.js
9:57:48 PM: src/schema/types/species.js -> lib/schema/types/species.js
9:57:48 PM: src/schema/types/starship.js -> lib/schema/types/starship.js
9:57:48 PM: src/schema/types/vehicle.js -> lib/schema/types/vehicle.js
9:57:48 PM: src/server.js -> lib/server.js
9:57:48 PM: src/service.js -> lib/service.js
9:57:48 PM: src/swapi-rest.js -> lib/swapi-rest.js
9:57:48 PM: > swapi-graphql@ build:lambda /opt/build/repo
9:57:48 PM: > NODE_ENV=development netlify-lambda build handler
9:57:49 PM: netlify-lambda: Building functions
9:57:49 PM: ModuleNotFoundError: Module not found: Error: Can't resolve '../cache/data.json' in '/opt/build/repo/lib'    at factory.create (/opt/build/repo/node_modules/webpack/lib/Compilation.js:925:10)
9:57:49 PM:     at factory (/opt/build/repo/node_modules/webpack/lib/NormalModuleFactory.js:401:22)
9:57:49 PM:     at resolver (/opt/build/repo/node_modules/webpack/lib/NormalModuleFactory.js:130:21)
9:57:49 PM:     at asyncLib.parallel (/opt/build/repo/node_modules/webpack/lib/NormalModuleFactory.js:224:22)
9:57:49 PM:     at /opt/build/repo/node_modules/neo-async/async.js:2830:7
9:57:49 PM:     at /opt/build/repo/node_modules/neo-async/async.js:6877:13
9:57:49 PM:     at normalResolver.resolve (/opt/build/repo/node_modules/webpack/lib/NormalModuleFactory.js:214:25)
9:57:49 PM:     at doResolve (/opt/build/repo/node_modules/enhanced-resolve/lib/Resolver.js:213:14)
9:57:49 PM:     at hook.callAsync (/opt/build/repo/node_modules/enhanced-resolve/lib/Resolver.js:285:5)
9:57:49 PM:     at _fn0 (eval at create (/opt/build/repo/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:15:1)
9:57:49 PM: resolve '../cache/data.json' in '/opt/build/repo/lib'
9:57:49 PM:   using description file: /opt/build/repo/package.json (relative path: ./lib)
9:57:49 PM:     using description file: /opt/build/repo/package.json (relative path: ./cache/data.json)
9:57:49 PM:       no extension
9:57:49 PM:         /opt/build/repo/cache/data.json doesn't exist      .wasm
9:57:49 PM:         /opt/build/repo/cache/data.json.wasm doesn't exist      .mjs
9:57:49 PM:         /opt/build/repo/cache/data.json.mjs doesn't exist      .js
9:57:49 PM:         /opt/build/repo/cache/data.json.js doesn't exist      .json
9:57:49 PM:         /opt/build/repo/cache/data.json.json doesn't exist      .ts
9:57:49 PM:         /opt/build/repo/cache/data.json.ts doesn't exist      as directory
9:57:49 PM:         /opt/build/repo/cache/data.json doesn't existnpm ERR! code ELIFECYCLE
9:57:49 PM: npm ERR! errno 1
9:57:49 PM: npm ERR! swapi-graphql@ build:lambda: `NODE_ENV=development netlify-lambda build handler`
9:57:49 PM: npm ERR! Exit status 1
9:57:49 PM: npm ERR!
9:57:49 PM: npm ERR! Failed at the swapi-graphql@ build:lambda script.
9:57:49 PM: npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
9:57:49 PM: npm ERR! A complete log of this run can be found in:
9:57:49 PM: npm ERR!     /opt/buildhome/.npm/_logs/2020-10-17T18_57_49_859Z-debug.log
9:57:49 PM: npm ERR! code ELIFECYCLE
9:57:49 PM: npm ERR! errno 1
9:57:49 PM: npm ERR! swapi-graphql@ build: `rimraf lib && babel src --ignore __tests__,public --out-dir lib/ && npm run build:lambda`
9:57:49 PM: npm ERR! Exit status 1
9:57:49 PM: npm ERR!
9:57:49 PM: npm ERR! Failed at the swapi-graphql@ build script.
9:57:49 PM: npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
9:57:49 PM: npm ERR! A complete log of this run can be found in:
9:57:49 PM: npm ERR!     /opt/buildhome/.npm/_logs/2020-10-17T18_57_49_880Z-debug.log
9:57:49 PM: ​
9:57:49 PM: ┌─────────────────────────────┐
9:57:49 PM: │   "build.command" failed    │
9:57:49 PM: └─────────────────────────────┘
9:57:49 PM: ​
9:57:49 PM:   Error message
9:57:49 PM:   Command failed with exit code 1: npm run build
9:57:49 PM: ​
9:57:49 PM:   Error location
9:57:49 PM:   In build.command from netlify.toml:
9:57:49 PM:   npm run build
9:57:49 PM: ​
9:57:49 PM:   Resolved config
9:57:49 PM:   build:
9:57:49 PM:     command: npm run build
9:57:49 PM:     commandOrigin: config
9:57:49 PM:     environment:
9:57:49 PM:       - REVIEW_ID
9:57:49 PM:     functions: /opt/build/repo/netlify-lambda-lib
9:57:49 PM:     publish: /opt/build/repo/public
9:57:50 PM: Caching artifacts
9:57:50 PM: Started saving node modules
9:57:50 PM: Finished saving node modules
9:57:50 PM: Started saving build plugins
9:57:50 PM: Finished saving build plugins
9:57:50 PM: Started saving pip cache
9:57:50 PM: Finished saving pip cache
9:57:50 PM: Started saving emacs cask dependencies
9:57:50 PM: Finished saving emacs cask dependencies
9:57:50 PM: Started saving maven dependencies
9:57:50 PM: Finished saving maven dependencies
9:57:50 PM: Started saving boot dependencies
9:57:50 PM: Finished saving boot dependencies
9:57:50 PM: Started saving go dependencies
9:57:51 PM: Finished saving go dependencies
9:57:51 PM: Build failed due to a user error: Build script returned non-zero exit code: 2
9:57:51 PM: Failing build: Failed to build site
9:57:51 PM: Failed during stage 'building site': Build script returned non-zero exit code: 2
9:57:51 PM: Finished processing build request in 26.682442851s
</pre>
</details>

--- a/netlify.toml
+++ b/netlify.toml
@@ -1,3 +1,3 @@
 [build]
   functions = "netlify-lambda-lib"
-  command = "npm run build"
+  command = "npm run download && npm run build"

